util/EIP-1024/README.md

```diff
@@ -0,0 +1,51 @@
+# EIP-1024 ![Crates.io](https://img.shields.io/crates/d/eip-1024.svg) [![Released API docs](https://docs.rs/EIP-1024/badge.svg)](https://docs.rs/EIP-1024)
+
+## Example
+
+```rust
+use eip_1024;
+use saltbabe::{KeyPair,Public, Secret, Error};
+use saltbabe::traits::FromUnsafeSlice;
+
+fn main() {
+	let bob_sk = "mJxmrVq8pfeR80HMZBTkjV+RiND1lqPqLuCdDUiduis=";
```

Comment:
This line is correctly indented using tabs, the rest uses spaces. Please reformat to all tabs.

Reply:
ok